tests/lib/helpers.ts

```diff
@@ -0,0 +1,101 @@
+/// <reference path="../data/metrics.ts" />
+
+interface Window {
+    jQuery: JQueryStatic
+    QUnit: QUnit
+}
+
+class TestHelper {
```

Comment:
It appears that the [Karma html2js preprocessor](https://github.com/karma-runner/karma-html2js-preprocessor) allows fixtures to be loaded directly by Karma. Does it fit the use-case?